app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  validates :body, presence: true
+  validates :body, length: { minimum: 4 }
```

Comment:
You can write like this :
`validates :body, presence: true,  length: { minimum: 4  }`